conf/cmi/group.role.upper_secondary_school-97a902313.yml

```diff
@@ -0,0 +1,17 @@
+uuid: 81cf1f11-3038-4208-899b-5df33f67ff16
+langcode: en
+status: true
+dependencies:
+  config:
+    - group.type.upper_secondary_school
+  enforced:
+    config:
+      - user.role.read_only
+id: upper_secondary_school-97a902313
+label: 'Read only'
```

Comment:
I wonder if these roles will be broken now that the dev-branch has the role changes (new roles & changes for names)

Reply:
Checking and adding permissions regarding new roles.